src/utils.h

```diff
@@ -35,41 +38,53 @@ extern log_level_t log_level;
 extern char *log_cid;
 extern gboolean use_syslog;
 
+static inline const char *get_errno_string(void)
+{
+	static __thread char errbuf[256];
```

Comment:
(Absolutely unblocking: The `__thread` works but it seems unnecessary, the caller can allocate a buffer on stack.)